test/unit/lib/addMessages.js

```diff
@@ -7,6 +7,7 @@ const addMessages = require('../../../app/lib/addMessages');
 const expect = chai.expect;
 
 describe('addMessages', () => {
+  const nowDateString = new Date().toDateString();
```

Comment:
It might be better to use a specific time and moment in the tests now that nextOpen is being asserted, rather than use `new Date()` and `moment()` to get the current time.

Reply:
Fair point, I'll make the changes.